<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 202, 255);
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>419</width>
      <height>729</height>
     </rect>
    </property>
    <widget class="QPushButton" name="FlightsButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Flights</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RestaurantsButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Restaurants</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TransportButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Tranportation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AttractionsButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Attractions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ForumsButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Forums</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HomeButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>670</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>670</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ProfileButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>670</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="FavoritesButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>670</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Favorites</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="e-ViVa">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Forte</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(162, 202, 255);
color: rgb(27, 36, 156);

</string>
     </property>
     <property name="text">
      <string>e-ViVa</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>381</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 228, 232);
color: rgb(41, 75, 117);


</string>
     </property>
     <item>
      <property name="text">
       <string>Nearby Activities</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="AccomodationButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Accomodation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>361</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(41, 75, 117);
background-color: rgb(206, 255, 217);</string>
     </property>
     <property name="text">
      <string>Activity 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>361</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(41, 75, 117);
background-color: rgb(206, 255, 217);</string>
     </property>
     <property name="text">
      <string>Activity 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>530</y>
       <width>361</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(41, 75, 117);
background-color: rgb(206, 255, 217);</string>
     </property>
     <property name="text">
      <string>Maps</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAccomodation">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hotels</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Motels</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Air-Bnb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Camping</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxAttractions">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Museums</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Concerts</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxRestaurants">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cafe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fast Food</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fine Dinning</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxTransport">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(48, 76, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Taxi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uber</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Metro</string>
      </property>
     </item>
    </widget>
    <zorder>ProfileButton</zorder>
    <zorder>ForumsButton</zorder>
    <zorder>FavoritesButton</zorder>
    <zorder>SearchButton</zorder>
    <zorder>HomeButton</zorder>
    <zorder>RestaurantsButton</zorder>
    <zorder>TransportButton</zorder>
    <zorder>FlightsButton</zorder>
    <zorder>AttractionsButton</zorder>
    <zorder>e-ViVa</zorder>
    <zorder>listWidget</zorder>
    <zorder>AccomodationButton</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>comboBoxAccomodation</zorder>
    <zorder>comboBoxAttractions</zorder>
    <zorder>comboBoxRestaurants</zorder>
    <zorder>comboBoxTransport</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
